src/electionguard-ui/ElectionGuard.UI/Models/TallyCeremonyChecklist.cs

```diff
@@ -2,7 +2,9 @@
 
 public partial class TallyCeremonyChecklist : ObservableObject
 {
-    public bool QuorumReached => _guardiansJoined >= _quorum;
+    [ObservableProperty]
+    private bool _quorumReached;
```

Comment:
Need to switch this to an actual property since it was throwing an exception that it was not being found.

Reply:
Good catch